src/index.js

```diff
@@ -5,26 +5,70 @@ import Traveler from './Traveler';
 import Destination from "./Destination";
 import Trip from './Trip';
 
-let traveler = new Traveler();
-let trip = new Trip()
-let destination = new Destination()
-let allDestinationsData = [];
-let allTripsData = [];
+let currentTraveler;
+let currentTravelerTrips;
+let currentTravelerDestinations
+let allDestinationsData;
+let allTripsData;
```

Comment:
You could refactor lines 8-12 here by having one line with each empty variable by writing `let currentTraveler, currentTravelerTrips, currentTravelerDestinations, ...etc`

Reply:
good call! Totally forgot about that